cabby/geo/walk.py

```diff
@@ -119,12 +119,16 @@ def get_start_poi(map: map_structure.Map, end_point: GeoDataFrame) -> \
     outer_circle_graph = ox.truncate.truncate_graph_dist(
         map.nx_graph, dest, max_dist=2000, weight='length')
 
-    # Get graph that is too close (less than 400 meter path distance)
-    inner_circle_graph = ox.truncate.truncate_graph_dist(
-        map.nx_graph, dest, max_dist=400, weight='length')
-
     outer_circle_graph_osmid = list(outer_circle_graph.nodes.keys())
-    inner_circle_graph_osmid = list(inner_circle_graph.nodes.keys())
+
+    try:
+        # Get graph that is too close (less than 400 meter path distance)
+        inner_circle_graph = ox.truncate.truncate_graph_dist(
+            map.nx_graph, dest, max_dist=400, weight='length')
+        inner_circle_graph_osmid = list(inner_circle_graph.nodes.keys())
+
+    except:
```

Comment:
Need to declare an explicit exception -- see https://google.github.io/styleguide/pyguide.html#24-exceptions

It may be better to use some other means to capture the behavior, but it's hard based on the current code to know what errors might arise and thus hard to suggest other alternatives. Can you also add some documentation?

Reply:
Doesn't look like you did, e.g. the code currently says:

```
    try:
        # Get graph that is too close (less than 400 meter path distance)
        inner_circle_graph = ox.truncate.truncate_graph_dist(
            map.nx_graph, dest, max_dist=400, weight='length')
        inner_circle_graph_osmid = list(inner_circle_graph.nodes.keys())

    except:
        inner_circle_graph_osmid = []
```

Did you forget to push it?

Reply:
Yes I forgot